src/write_http.c

```diff
@@ -468,9 +471,9 @@ static int wh_write_kairosdb(const data_set_t *ds,
     }
   }
 
-  status = format_kairosdb_value_list(cb->send_buffer, &cb->send_buffer_fill,
-                                      &cb->send_buffer_free, ds, vl,
-                                      cb->store_rates);
+  status = format_kairosdb_value_list(
+      cb->send_buffer, &cb->send_buffer_fill, &cb->send_buffer_free, ds, vl,
+      cb->store_rates, (char const *const *)http_attrs, http_attrs_num);
```

Comment:
`src/write_http.c: In function ‘wh_write_kairosdb’:
src/write_http.c:477:24: error: passing argument 7 of ‘format_kairosdb_value_list’ from incompatible pointer type [-Werror=incompatible-pointer-types]
       cb->store_rates, http_attrs, http_attrs_num);
                        ^~~~~~~~~~
In file included from src/write_http.c:31:0:
src/utils_format_kairosdb.h:40:5: note: expected ‘const char * const*’ but argument is of type ‘char **’
 int format_kairosdb_value_list(char *buffer, size_t *ret_buffer_fill,
     ^~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors`